CHANGES.rst

```diff
@@ -9,6 +9,8 @@
 
 - Add support for Python 3.9.
 
+- Fix FieldUpdateEvent implementation by having an `object` attribute as the
+  `IFieldUpdatedEvent` interfaces claims there should be.
```

Comment:
```suggestion
- Fix FieldUpdateEvent implementation by having an ``object`` attribute as the
  ``IFieldUpdatedEvent`` interfaces claims there should be.
```

One backtick is for the "default role", which could mean anything. Two backticks is for preformatted code.